annotation/util.scm

```diff
@@ -291,7 +292,7 @@
  (string-append "https://www.ncbi.nlm.nih.gov/pubmed/?term=" (cadr (string-split nodename #\:)))
 )
 
-(define-public (write-to-file result id name)
+(define* (write-to-file result id name #:optional (ext ".scm"))
```

Comment:
There is a way of doing `define*-public` one of the ice-9 modules provides this, I forget which.

Reply:
are you referring to `(ice-9 optargs)`?

Reply:
yes. But looking at your code, it seems you don't need it, if you have an explicit export statement. (that is, you only need to say `export` or `define-public`, you don't need both.) So whatever. coding trivia.